api/v1/checluster_types.go

```diff
@@ -360,6 +361,12 @@ type CheClusterSpecServer struct {
 	WorkspacePodNodeSelector map[string]string `json:"workspacePodNodeSelector,omitempty"`
 	// The pod tolerations put on the workspace pods to limit where the workspace pods can run.
 	WorkspacePodTolerations []corev1.Toleration `json:"workspacePodTolerations,omitempty"`
+	// The default editor to workspace create with.
+	// +optional
+	WorkspaceDefaultEditor string `json:"workspaceDefaultEditor,omitempty"`
+	// The default components to workspace create with.
+	// +optional
+	WorkspaceDefaultComponents []devfile.BaseComponent `json:"workspaceDefaultComponents,omitempty"`
```

Comment:
Are these default components meant to be used when a Devfile does not contain any components, or are they added too all Devfiles? We should document this.